getting_started/dev-environment/macOS.md

```diff
@@ -0,0 +1,65 @@
+# Instructions for macOS
+
+Install required software:
+- [Homebrew](http://brew.sh/)
+- [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+- [Vagrant](https://www.vagrantup.com/downloads.html)
+
+Install some basic command line tools by running:
+
+```shell
+$ brew install coreutils
+```
+
+Set your PATH to use coreutils:
+
+```shell
+export PATH=/usr/local/opt/coreutils/libexec/gnubin:/usr/local/sbin:$PATH
+export MANPATH=/usr/local/opt/coreutils/libexec/gnuman:$MANPATH
+```
+
+Install the python package manager pip:
+
+```shell
+$ curl -O https://raw.github.com/pypa/pip/master/contrib/get-pip.py
+$ python get-pip.py
+$ rm get-pip.py
+```
+
+Install setuptools, virtualenv and virtualenvwrapper:
+
+```shell
+$ sudo pip install setuptools virtualenv virtualenvwrapper
+```
+
+Add the following to your `~/.bash_profile`:
+
+```shell
+export WORKON_HOME=~/virtualenvs
+source /usr/local/bin/virtualenvwrapper.sh
+```
+
+Enable virtualenvwrapper:
+
+```shell
+$ source ~/.bash_profile
+```
+
+Install [vagrant-hostmanager](https://github.com/devopsgroup-io/vagrant-hostmanager), and [vagrant-triggers](https://github.com/emyl/vagrant-triggers):
+
+```shell
+$ vagrant plugin install vagrant-hostmanager
+$ vagrant plugin install vagrant-triggers
```

Comment:
These can be done as one line:
```markdown
    ```shell
    vagrant plugin install vagrant-hostmanager vagrant-triggers
    ```
```